server/src/resolvers/datetimeResolver.ts

```diff
@@ -0,0 +1,7 @@
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+const customScalarResolver = {
+  DateTime: GraphQLDateTime
+};
+
+export default customScalarResolver;
```

Comment:
I think I imported a different datetime scalar from graphql-tools, we should probably use the same one

Reply:
Honestly I don't know the difference and they probably do the exact same thing. I think that all of our graphql dependencies are coming from graphql-tools at the moment tho so I'm leaning towards that one